show_notes/Episode_16.md

```diff
@@ -0,0 +1,39 @@
+# Episode 16: Tools
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/86)
+
+In this episode Artem, Artur and Hannes discuss the tools they use for development.
+
+#### Hosts:
+
+ - Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](http://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+  - Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+  - Artur Dryomov [@arturdryomov](https://twitter.com/arturdryomov), [personal blog](http://arturdryomov.online/), [GitHub](https://github.com/ming13)
+  
+#### Links:
+
+Tools:
+
+  - [Mainframer](https://github.com/gojuno/mainframer)
+  - [Composer](https://github.com/gojuno/composer)
+  - [Swarmer](https://github.com/gojuno/swarmer)
+  - [zsh](http://ohmyz.sh)
+  - [cloc](https://github.com/AlDanial/cloc)
+  
+CI:
+  
+  - [Jenkins](https://jenkins.io)
+  - [Circle.Ci](http://circleci.com)
+  - [Drone](https://drone.io)
+
+
+Libraries:
+
+  - [Spek](http://spekframework.org/)
+  - [RxReplayingShare](https://github.com/JakeWharton/RxReplayingSharel)
```

Comment:
XML comments can be applied here as well.

Reply:
What do you mean?